examples/tv-casting-app/android/App/app/src/main/AndroidManifest.xml

```diff
@@ -19,9 +19,9 @@
         android:label="@string/app_name"
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
-        android:name=".ChipTvCastingApplication"
+        android:name=".chip.casting.app.ChipTvCastingApplication"
```

Comment:
On line 13 of this file CHANGE\_WIFI\_MULTICAST\_STATE permission is listed a second time (listed first at line 9). Can you delete line 13?